### ISSUE TITLE
update prompt doesn't say what is being updated

### ISSUE THREAD

**Issue by benlk** [wp-cli/wp-cli#5107]
(This issue should be moved to the appropriate component repo if this functionality is not part of `wp-cli`'s core functionality, and closed if this has been fixed in newer versions of wp-cli.)

Consider the following command-line output from wp-cli 2.0.1:

```
$ wp help plugin install
You have version 2.0.1. Would you like to update to 2.1.0? [y/n]
```

The user appears to be running a `help` command. Why would wp-cli be asking to update an undefined something?

In this message:
- make it clear that this message is from `wp-cli`, not something that may have hijacked wp-cli's `help` function
- make it clear that `wp-cli` is asking to upgrade `wp-cli`, not a plugin or something else

**Comment by schlessera** [comment]
Yes, that makes total sense. Would you be up for working on a PR to improve this message?

**Comment by benlk** [comment]
Sure. Is that limited to these strings?

https://github.com/wp-cli/wp-cli/blob/cd049d2cedef57106160c5f37f9caa14bcce0d7c/php/commands/src/CLI_Command.php#L287-L307

**Comment by danielbachhuber** [comment]
> Is that limited to these strings?

Yep, just those strings.

**Comment by karthick-murugan** [comment]
Hi @benlk.,

Is this issue still valid, because when I run this command `wp help plugin install` now I can see only the help screen as shown in this screenshot.

![Image](https://github.com/user-attachments/assets/132b4247-a198-49a6-871c-4193a38f5b50)
